Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -49,9 +49,9 @@ final class TensorAutoDiffTests: XCTestCase {
             }
             return y.0 + y.1 - z.0.0 + z.0.1
         }
-        XCTAssertTrue((value: Tensor(8), gradient: Tensor(2)) == valueWithGradient(at: Tensor(4), in: condNestedTupleVar))
-        XCTAssertTrue((value: Tensor(-20), gradient: Tensor(2)) == valueWithGradient(at: Tensor(-10), in: condNestedTupleVar))
-        XCTAssertTrue((value: Tensor(-2674), gradient: Tensor(2)) == valueWithGradient(at: Tensor(-1337), in: condNestedTupleVar))
+        XCTAssertTrue((value: Tensor<Float>(8), gradient: Tensor<Float>(2)) == valueWithGradient(at: Tensor<Float>(4), in: condNestedTupleVar))
+        XCTAssertTrue((value: Tensor<Float>(-20), gradient: Tensor<Float>(2)) == valueWithGradient(at: Tensor<Float>(-10), in: condNestedTupleVar))
+        XCTAssertTrue((value: Tensor<Float>(-2674), gradient: Tensor<Float>(2)) == valueWithGradient(at: Tensor<Float>(-1337), in: condNestedTupleVar))
```

Comment:
These lines are over the 100 column limit, but I'm not too concerned because we may look into using [`swift-format`](https://github.com/apple/swift-format) to auto-format the ~entire repository soon, now that SR-12002 has been fixed.